docs/notebooks/4.1 Setting up and Using PV Sites.ipynb

```diff
@@ -0,0 +1,521 @@
+      "text/plain": [
+       "<Figure size 1500x400 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# NOTE: this will consume an API request!\n",
```

Comment:
Add general note at the top of the example that some of the requests below do use API requests but that if they want to test the advanced pv power live and forcast endpoints without consuming API requests they can used the unmetered locations as found in `unmetered_locations` and as shown

Reply:
done